cypress/integration/plan.visit.spec.js

```diff
@@ -0,0 +1,98 @@
+import address from "../fixtures/address.json"
+import visit from "../fixtures/visit.json"
+
+describe('Plan "huisbezoek"', () => {
+
+  describe('Go to "Bezoek inplannen" form', () => {
+
+    it("Login as projectmedewerker", () => {
+      cy.loginAsPm()
+    })
+
+    it("Go to Adresoverzicht", () => {
+      cy.visit(`/adres/${address.bagId}`)
+      cy.wait(2000)
+    })
+
+    it("Adresoverzicht has right address", () => {
+      cy.get("h1")
+        .contains(`${address.street}, ${address.zipCode}`)
+    })
+
+    it('Get first case with task "Inplannen Huisbezoek"', () => {
+      cy.get("tbody>tr")
+        .contains("td", "Inplannen Huisbezoek")
+        .click()
+    })
+
+    it("Click on task Bezoek inplannen", () => {
+      cy.get("tbody>tr")
+        .contains("td", "Bezoek inplannen")
+        .click()
+    })
+
+    it("Bezoek inplannen page is visible", () => {
+      cy.get("h1")
+        .contains("Bezoek inplannen")
+
+      cy.get("dd")
+        .contains(address.street)
+    })
+  })
+
+  describe('Fill in "Bezoek inplannen" form', () => {
+
+    it("Schedule day", () => {
+      cy.get('[data-e2e-id="week_segment"]')
+        .should("be.visible")
```

Comment:
`.should('be.visible')` is often unnecessary in forms, see https://iheartjs.dev/cypress-tip-dont-overuse-the-visibility-assertion/